apps/meteor/app/lib/server/lib/notifyUsersOnMessage.ts

```diff
@@ -148,13 +126,14 @@ export async function updateThreadUsersSubscriptions(message, room, replies) {
 
 	await Subscriptions.setOpenForRoomIdAndUserIds(message.rid, repliesPlusSender);
 
-	await Subscriptions.setLastReplyForRoomIdAndUserIds(message.rid, repliesPlusSender, new Date());
+	// TODO: Fix this cause is waiting for a single string and it should (?!) be an array
+	await Subscriptions.setLastReplyForRoomIdAndUserIds(message.rid, repliesPlusSender as unknown as string, new Date());
 }
 
-export async function notifyUsersOnMessage(message, room) {
+export async function notifyUsersOnMessage(message: IMessage & { editedAt?: Date | undefined }, room: IRoom): Promise<IMessage> {
```

Comment:
I think you can use the method, isEditedMessage from IMessage, so we can hide this type gimnastics in the model and don't have to expose it

Reply:
Now using `IEditedMessage` instead of `IMessage`. https://github.com/RocketChat/Rocket.Chat/pull/31877/commits/359e5dc9816d42e8cfe37721ecf1a7df76e9f27c

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/31877/commits/5f39dfc565f70291947bda7d7badcd8130c26477